css/menu.css

```diff
@@ -298,3 +298,12 @@ body.disableScroll {
 .hide-scroll {
   overflow: hidden;
 }
+
+/* IE11 layout */
+@media screen and (-ms-high-contrast: none) {
+  .fl-bottom-bar-menu-holder.expanded,
+  [data-has-notch] .fl-bottom-bar-menu-holder.expanded,
+  .fl-bottom-bar-menu-holder.expanded ul {
+    height: auto;
```

Comment:
You are overriding the changes made in the PR https://github.com/Fliplet/fliplet-menu-bottom-bar/pull/22, which sets the height to 100% in order to fix this issue https://github.com/Fliplet/fliplet-studio/issues/4828

Can you confirm that by reverting this on IE 11 you haven't recreated the issue https://github.com/Fliplet/fliplet-studio/issues/4828?

Reply:
@tonytlwu 
I have retested 4828 on IE and everything seems to work correctly.
![ie-menu](https://user-images.githubusercontent.com/52824207/80792421-1ebc3a80-8b9d-11ea-8998-7e1580343b81.gif)

Reply:
👍